R/roxygen.R

```diff
@@ -24,11 +27,14 @@
 #'   boolean (indicating that all installed packages should be rebuilt), or a
 #'   vector of package names indicating which packages should be rebuilt.
 #'
-#' @param repos The repositories to use during restore, for packages installed
-#'   from CRAN or another similar R package repository. When set, this will
-#'   override any repositories declared in the lockfile. See also the
-#'   `repos.override` option in [config] for an alternate way to provide a
-#'   repository override.
+#' @param repos The repositories to use when restoring packages installed
+#'   from CRAN or a CRAN-like repository. By default, will use repositories
+#'   recorded in the lockfile, ensuring that CRAN packages are re-installed
+#'   from the same place.
+#'
+#'   Use `repos = getOptions(repos)` to overide with the repositories set
```

Comment:
But `snapshot()` defaults to `repos = getOption("repos")` already?

Reply:
This text is only used by `restore()` and `install()`

Reply:
Ah, got it. Thanks!